12_rpn_calculator/rpn_calculator.rb

```diff
@@ -0,0 +1,93 @@
+class RPNCalculator
+  def initialize
+    @stack = []
+    @total = 0
+  end
+
+  def push(number)
+    @stack << number
+  end
+
+  def value
+    @total
+  end
+
+  def plus
```

Comment:
Try extracting each `if` into a private method to clean this up.

You could then do something like:

```ruby
do_something if stack.length >=2
do_something_else if stack.length == 1
do_abc if stack.empty?
```